docs/modules/releasenotes/partials/release-notes-24.1.adoc

```diff
@@ -55,3 +55,38 @@ With JavaScript, you'll just subscribe for one or more topics and add a handler
 To publish a notification, just add any xref:technical-guide:working-with-data/data-object.adoc[Data Object] to the notification registry for a topic.
 
 Learn more about the usage and functionality in the chapter xref:technical-guide:working-with-data/ui-notifications.adoc[UI Notifications].
+
+== Apache HttpComponents Client 5.2.x
+
+With Scout version 7 the usage of Apache HttpComponents Client was introduced (for example for communication between frontend and backend server). At that time version 4.5.3 was used which received only minor updates between releases (for major defects and security issues), version 24.1 of Scout upgrades the Apache HTTP Client to the next major version - now version 5.2.x is used.
+
+Scout does not depend itself on an older Apache HttpComponents Client version anymore; however the new client version can be co-located with earlier major version on the same classpath as package names and maven coordinates have been changed.
+
+This may result in changes if your project references (imports) Apache HttpComponents classes itself, e.g. `org.apache.http.HttpHeaders` has been moved to `org.apache.hc.core5.http.HttpHeaders`, `org.apache.http.impl.client.HttpClientBuilder` has been moved to `org.apache.hc.client5.http.impl.classic.HttpClientBuilder`, for further details the link:https://hc.apache.org/httpcomponents-client-5.2.x/migration-guide/index.html[Apache HttpClient 5.x migration guide] may be consulted (at the moment the classic API is still used to avoid too many changes).
+
+Of course there is also behavior and API which has changed, the following behavioral changes have been noticed during development:
+
+* `org.eclipse.scout.rt.shared.http.EnhancedLaxRedirectStrategy` is removed, this strategy previously enforced a second HTTP POST request for certain HTTP redirect status codes (e.g. 302 Found). According to link:https://www.rfc-editor.org/rfc/rfc9110.html#status.302[RFC 9110] in this case the POST request may be replaced with a GET request which the HttpClient will now respect. For redirects which require to re-send the POST request a different status code must be used (e.g. 307 Temporary Redirect). The config property `scout.http.redirectPost` has been removed.
+* Internal test `org.eclipse.scout.rt.server.commons.http.HttpRetryTest.testPostWithUnsupportedRetryAndFailureWhileHeadersAreSent` has been removed as retry-strategy has been changed, requests failing with a `org.apache.hc.core5.http.ConnectionClosedException` while sending the request are not retried anymore in general, furthermore only idempotent requests will be retried by default. However, the Scout specific implementation to retry requests dealing with a stale socket channel is kept (regardless of the request method).
+
+For third-party dependencies which refer to an older HttpClient version either both versions will co-exist (see above) or it is necessary to search for updated third-party versions, e.g. the maven dependency
```

Comment:
I would skip this part, since the jersey-apache-connector should never be necessary when working with Scout. Since release 23 Scout provides its own implementation of an Jersey - Apache connector.

Reply:
Solved in amended commit (removed examples)